README.md

```diff
@@ -75,34 +103,18 @@ The following executables will be created in the `build/bin` subfolder:
 * `vcf_debugulator`: automatic fixing tool
 * `test_validator` and derivatives: testing correct behaviour of the tools listed above
 
-## Dynamic build
+#### Static build
 
-**Note:** Please ignore this section if you only want to use the application.
+### Mac OSX
 
-The end-users build is perfectly valid during development to generate a static binary. Please follow the instructions below if you would like to generate a dynamically linked binary.
 
-### Dependencies
-
-#### Boost
 
-The dependencies are the Boost library core, and its submodules: Boost.filesystem, Boost.program_options, Boost.regex, Boost.log and Boost.system.
-If you are using Ubuntu, the required packages' names will be `libboost-dev`, `libboost-filesystem-dev`, `libboost-program-options-dev`, `libboost-regex-dev` and `libboost-log-dev`.
-
-#### ODB
-
-You will need to download the ODB compiler, the ODB common runtime library, and the SQLite database runtime library from [this page](http://codesynthesis.com/products/odb/download.xhtml).
-
-ODB requires SQLite3 to be installed. If you are using Ubuntu, the required packages' names will be `libsqlite3-0` and `libsqlite3-dev`.
-
-To install the ODB compiler, the easiest way is to download the `.deb` or `.rpm` packages, in order to be installed automatically with `dpkg`. Both the ODB runtime and SQLite database runtime libraries can be installed manually running `./configure && make && sudo make install`. This will install the libraries in `/usr/local/lib`.
-
-If you don't have root permissions, please run `./configure --prefix=/path/to/odb/libraries/folder` to specify which folder to install ODB in, then `make && make install`, without `sudo`.
 
-### Compile
+## Compile
 
 The build has been tested on the following compilers:
-* Clang 3.5 to 3.7
-* GCC 4.8 to 5.0
+* Clang 3.5 to 5.0
+* GCC 4.8 to 7.0
```

Comment:
Agreed offline to refer only to the versions listed on Travis, meaning Clang 3.9 to 5.0 and GCC 4.8 to 6.0.